src/main/java/webserver/HttpHeaders.java

```diff
@@ -0,0 +1,41 @@
+package webserver;
+
+import java.util.*;
+
+public class HttpHeaders {
+
+    private Map<String, List<String>> loweredKeyValueListMap = new HashMap<>();
+
+    public void addHeader(String k, String v) {
+        loweredKeyValueListMap.compute(k.toLowerCase(), (key, vals) -> {
+            if (vals == null) {
+                vals = new ArrayList<>();
+            }
+
+            vals.add(v);
+            return vals;
+        });
+    }
+
+    public void addHeaders(HttpHeaders otherHeaders) {
+        for (Map.Entry<String, List<String>> entry : otherHeaders.entrySet()) {
+            for (String value : entry.getValue()) {
+                addHeader(entry.getKey(), value);
+            }
+        }
```

Comment:
```suggestion
        otherHeaders.forEach((name, headerValues) -> {
            headerValues.forEach(value -> addHeader(name, value));
        });
```
[도전] 요런식으로 동작할 수 있는 `HttpHeaders#forEach`를 만들어 보세요. 😀